pyteal/ast/subroutine_test.py

```diff
@@ -23,10 +23,10 @@ def fn2Args(a1, a2):
     def fn10Args(a1, a2, a3, a4, a5, a6, a7, a8, a9, a10):
         return Return()
 
-    lam0Args = lambda: Return()
-    lam1Args = lambda a1: Return()
-    lam2Args = lambda a1, a2: Return()
-    lam10Args = lambda a1, a2, a3, a4, a5, a6, a7, a8, a9, a10: Return()
+    lam0Args = lambda: Return()  # noqa: E731
```

Comment:
@tzaffi If you prefer, I'm up for ignoring E731 in tests rather than providing inline ignores?

It feels like the kind of check that we'll repeatedly run into during typical development cycles.  I don't feel strongly and am happy to resolve as is.

Reply:
I believe we should follow python's modern conventions and _not_ create lambda variables where functions work just as well

Reply:
Got it - That also works for me.  I had briefly wondered _why_ the style of conflicting definition is needed.

For the purpose of the PR, do you prefer to keep as is or make changes to remove the need for any inline ignores?

I'll go with either approach.

Reply:
I prefer to explicit ignore. That way I'm not messing with other people's code and also it can be a useful example of how we might want to ignore such situations in the future.